db/migrations/0001_initial.py

```diff
@@ -33,17 +52,16 @@ class Migration(migrations.Migration):
             name='Genre',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('name', models.CharField(max_length=255, unique=True)),
+                ('name', models.CharField(max_length=255)),
             ],
         ),
         migrations.CreateModel(
             name='Movie',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('title', models.CharField(max_length=255)),
+                ('title', models.CharField(max_length=255, unique=True)),
                 ('description', models.TextField()),
                 ('actors', models.ManyToManyField(to='db.Actor')),
```

Comment:
The `actors` field in the `Movie` model references `db.Actor`. It should reference `Actor` directly, as Django automatically resolves model references within the same app.